disperser/batcher/batcher.go

```diff
@@ -420,6 +420,15 @@ func (b *Batcher) HandleSingleBatch(ctx context.Context) error {
 	stageTimer = time.Now()
 	update := b.Dispatcher.DisperseBatch(ctx, batch.State, batch.EncodedBlobs, batch.BatchHeader)
 	log.Debug("DisperseBatch took", "duration", time.Since(stageTimer))
+	h, err := batch.State.OperatorState.Hash()
```

Comment:
The state here is for all operators, but the one used at node is for the specific operator, they may not be comparable

Reply:
The state here has operator states for all quorums, so it will produce hashes for operators states in all quorums (note that one operator state hash is calculated for each quorum)
The operator state at the node will contain operator states for quorums the operator is part of. So it will only return hashes for operator states the operator is part of. 
Each hash for each quorum should be comparable.